src/lib/listeners/ProgressEventListener.ts

```diff
@@ -8,7 +8,7 @@ import {Display} from '../Display';
 import {BundleName, getMessage} from '../messages';
 
 export interface ProgressEventListener {
-	listen(codeAnalyzer: CodeAnalyzer): void;
+	listen(...codeAnalyzers: CodeAnalyzer[]): void;
```

Comment:
Switching to `...` notation meant all the existing invocations can remain unchanged.